chart/kubeapps/values.yaml

```diff
@@ -729,6 +729,13 @@ apprepository:
   ##     ociRepositories:
   ##       - nginx
   ##       - jenkins
+  ##     # Optionally filter out some charts
+  ##     # Use variables so that the UI is able parse the expression
```

Comment:
Thanks @dud225 . Can you explain what you mean here? Haven't checked, but didn't think the UI was presenting the filter rules atm. This was just if you manually create/update an `AppReository` via the CLI, and you're adding here the ability to specify a filter rule in the config (great). Does the [example in the docs](https://github.com/kubeapps/kubeapps/blob/master/docs/user/private-app-repository.md#advanced-filtering) (which does not use vars) cause the UI to bail with a parsing error? (`    jq: .maintainers | any(.name == "Bitnami") `. ie. does `jq: .name == "nginx" or .name == "jenkins"` cause a parsing error?

Otherwise, +1 from me (sorry about CI, I'm hoping to get to that today).

Reply:
I should have read your related issue first. But yes, that's a bug if the previously defined filter is cleared. Still unsure about the above comment regarding variables.

Reply:
Hello
Yes sorry my comment is a bit blurry and doesn't actually relate to https://github.com/kubeapps/kubeapps/issues/3447.

I noticed that creating an AppRepository with filters set from the UI (I remember that the UI already presented them in v2.3.3):
![Capture d’écran de 2021-09-21 09-09-26](https://user-images.githubusercontent.com/17550475/134127107-ba36ca25-3e3e-4731-89e8-de4200eaa1fd.png)
yields the following rule:
```yaml
  filterRule:
    jq: .name == $var0 or .name == $var1
    variables:
      $var0: nginx
      $var1: jenkins
```
but if I edit it to just look as follows:
```yaml
  filterRule:
    jq: .name == "nginx" or .name == "jenkins"
```
then after refreshing Kubeapps the filter vanishes from the UI:
![Capture d’écran de 2021-09-21 09-13-12](https://user-images.githubusercontent.com/17550475/134127494-3d6f28c8-2c17-4f8a-bd07-d55363e71c3e.png)
even though the filter is sent by the server:
![Capture d’écran de 2021-09-21 09-21-50](https://user-images.githubusercontent.com/17550475/134128646-17cb36c0-2826-4652-be68-d08530b87f20.png)

Reverting the change makes the filter to appear again, so Kubeapps is definitely expecting some particular format in order for the UI to present the filter. 

So I may be missing something or this may be due to the fact that the UI can't support all variations of the filter rule, however I believe that some insights should be provided to the users to allow them to keep the UI in-sync with the CRD.

Reply:
Right, thanks. That gives me more context of the comment. Could you pls update the comment to something like: "The jq query format is not exposed in the UI, so care needs to be taken to use the format which the UI expects to parse, which is why variables are used in the example below."

Also, can you please merge or rebase with master (there was an issue with CI) then this should pass and I'm happy to land. Thanks for the improvement!

Reply:
Done. Thanks for your suggestion that clarifies my comment.